sys/config/src/config_store.c

```diff
@@ -67,7 +67,38 @@ conf_dst_register(struct conf_store *cs)
 static void
 conf_load_cb(char *name, char *val, void *cb_arg)
 {
-    conf_set_value(name, val);
+    if ((cb_arg && !strcmp((char*)cb_arg, name)) ||
```

Comment:
if (!cb_arg || !strcmp(cb_arg, name)) {

Reply:
Thanks, that's a neat way, did not think about it.